packages/google_sign_in/google_sign_in/example/lib/main.dart

```diff
@@ -59,7 +63,9 @@ class _SignInDemoState extends State<SignInDemo> {
       bool isAuthorized = account != null;
       // However, in the web...
       if (kIsWeb && account != null) {
+// #docregion CanAccessScopes
```

Comment:
Let's put this just above line 62 instead; it's all relevant to what the example is talking about.